pyigm/cgm/utils.py

```diff
@@ -84,6 +84,99 @@ def calc_rho(galaxy, igm_sys, cosmo, ang_sep=None, correct_lowz=True,
     return rho, ang_sep
 
 
+def get_close_galaxies(field,rholim=300.*u.kpc,minz=0.001,maxz=None):
+    '''
+    Generate table of galaxies close to sightlines for one more IgmGalaxyField
```

Comment:
one *or* more  ?

Reply:
typo -- should be '...for an IgmGalaxyField...'